contentcuration/contentcuration/frontend/channelEdit/vuex/contentNode/actions.js

```diff
@@ -338,23 +395,80 @@ export function deleteContentNodes(context, contentNodeIds) {
   );
 }
 
-export function copyContentNodes(context, contentNodeIds) {
-  // TODO: Implement copy nodes endpoint
-  return new Promise(resolve => resolve(context, contentNodeIds));
+export function copyContentNode(
+  context,
```

Comment:
There's definitely a limit for how many we can copy in a single sync endpoint request. I only encountered issues when trying to copy hundreds of nodes, mostly because of the AJAX timeout